src/main/java/se/sundsvall/notes/service/mapper/NoteMapper.java

```diff
@@ -46,6 +47,7 @@ public static NoteEntity toNoteEntity(NoteEntity noteEntity, UpdateNoteRequest u
 		ofNullable(updateNoteRequest.getCaseType()).ifPresent(noteEntity::setCaseType);
 		ofNullable(updateNoteRequest.getCaseLink()).ifPresent(noteEntity::setCaseLink);
 		ofNullable(updateNoteRequest.getExternalCaseId()).ifPresent(noteEntity::setExternalCaseId);
+		noteEntity.setMunicipalityId(municipalityId);
```

Comment:
Is this needed? When created the muncipalityId is set/stored - but when updated, the value is already set and will only be set to same id (if no bug has been introduced in service or api-layer), right? With that said, set of municipalityId can be removed.